di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/handlers/JwksHandler.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.di.ipv.stub.orc.handlers;
+
+import com.nimbusds.jose.jwk.ECKey;
+import com.nimbusds.jose.jwk.JWKSet;
+import io.javalin.http.Context;
+import uk.gov.di.ipv.stub.orc.utils.KeyIdGenerator;
+
+import java.util.List;
+
+import static com.nimbusds.jose.jwk.KeyUse.SIGNATURE;
+import static uk.gov.di.ipv.stub.orc.config.OrchestratorConfig.ORCHESTRATOR_CLIENT_ID;
+import static uk.gov.di.ipv.stub.orc.config.OrchestratorConfig.ORCHESTRATOR_SIGNING_JWK;
+
+public class JwksHandler {
+
+    public JwksHandler() {}
+
+    public void getResource(Context ctx) throws Exception {
+
+        ctx.json(
+                new JWKSet(
+                                List.of(
+                                        new ECKey.Builder(
+                                                        ECKey.parse(ORCHESTRATOR_SIGNING_JWK)
+                                                                .toPublicJWK())
+                                                .keyID(
+                                                        KeyIdGenerator.generate(
+                                                                ORCHESTRATOR_SIGNING_JWK,
+                                                                ORCHESTRATOR_CLIENT_ID))
```

Comment:
This will only list a signing key with kid used when the orch stub is operating as itself. We may in future want to include a key with the keyId used when acting as a stub for auth (mfa reset)

Reply:
I'm a bit less sure about this, we'll definitely need to support MFA reset... Given that we have a single orch stub, it's perhaps more realistic to just present a single consistent key on its JWKS endpoint? Or set up multiple different endpoints for orch vs. auth?

I'm also leaning towards configuring the `kid` in the JWK config itself - it feels less surprising than ignoring the `kid` there and generating a new one dynamically...

Reply:
I think the format seems reasonable - i.e. having a human-readable bit to make it easy to identify when copied around, but also a random/hashed part to avoid clashes.

I guess you're right about including multiple keys - for a slightly different reason than prod will be, but I guess it'll exercise the key-selection logic in core-back!